integration_tests/test_pull_properties.py

```diff
@@ -45,6 +45,24 @@ def assert_expected_pull_state(self, project_dir):
 
         # Verify that the contents of the dependencies made it in as well.
         self.assertTrue('foo' in state.properties)
+        self.assertTrue(len(state.assets['stage-packages']) > 0)
```

Comment:
The rest of this test actually verifies that the right thing is in here, this just verifies that there's _something_ there. That happens more explicitly in another test, but then why check anything here?